[Improved]: Use GroovyBaseScript's logging utility methods instead of using Debug (OFBIZ-11762)

priyasharma1 opened the pull request:
(OFBIZ-11762)

Explanation: 
- As per the discussion, updated all Debug.log* to log*
- Removed unused Debug import from the files
- Removed module variables as they are no longer useful.

Thanks: Pawan Verma

Commit:
[Improved]: Use GroovyBaseScript's logging utility methods instead of using Debug (OFBIZ-11762)

JacquesLeRoux commented:
Hi Priya,

Instead of 
logError(e, "")
why not simply using
logError(e)
?

priyasharma1 commented:
> Hi Priya,
> 
> Instead of
> logError(e, "")
> why not simply using
> logError(e)
> ?

In that case, we will have to overload logError once again, please confirm if this looks good to you, I will proceed accordingly.

Commit:
[Improved]: Updated all ocurances where we only needed to pass the throwable object, without a message
- We have now added one more utility method to handle these cases. (OFBIZ-11762)
